google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CreateInstanceRequest.java

```diff
@@ -35,7 +35,8 @@
  *       cluster. You cannot downgrade a production instance to a development instance.
  *   <dt>Development
  *   <dd>A low-cost instance for development and testing, with performance limited to the equivalent
- *       of a 1-node cluster. Development instances only support a single 1 node cluster.
+ *       of a 1-node cluster. Development instances only support a single 1 node cluster. At any
+ *       point this can be upgraded to production instance.
```

Comment:
a production instance

Reply:
Thanks, @elharo for the review!! Have updated JavaDoc accordingly.